apisix/plugins/cors.lua

```diff
@@ -202,9 +236,20 @@ local function set_cors_headers(conf, ctx)
     if conf.allow_credential then
         core.response.set_header("Access-Control-Allow-Credentials", true)
     end
+
+    if ctx.timing_allow_origin then
+        core.response.set_header("Timing-Allow-Origin", ctx.timing_allow_origin)
```

Comment:
Why do you need this?

Reply:
ok, useless, I may have forgotten to remove it when I decoupled the logic of classic cors and timing headers. I'll remove it